src/daft-dsl/src/resolve_expr/mod.rs

```diff
@@ -208,7 +208,7 @@ fn expand_wildcards(
 }
 
 /// Checks if an expression used in an aggregation is well formed.
```

Comment:
nit

```suggestion
/// Checks if an expression used in an aggregation is well-formed.

```
adding new line makes look better in rustdocs

https://rust-lang.github.io/rust-clippy/master/index.html#too_long_first_doc_paragraph